subcommittee/coding-guidelines/members.md

```diff
@@ -0,0 +1,11 @@
+# Members - Coding Guidelines Subcommittee
+
+- Walter Pearce
+- Joel Marcey
+- Alex Celeste
+- Florian Gilcher
+- Pete LeVasseur (Initial Chair)
+- Erik Jonkers
+- Jordan McQueen
+- Christof Petig
+- Alex Vochescu
```

Comment:
You can add me to that list; Daniel Glazman (Thales Group)

Reply:
Please add me as well: Sarah Dietrich (Festo)

Reply:
Me as well: Lukas Elmer (Siemens)

Reply:
Please also add me, Jonas Wolf (Vector Informatik)

Reply:
Also add Marc Schoolderman (Tweede Golf)